app/frontend/components/pages/sendAda/multiAssetsPage.tsx

```diff
@@ -0,0 +1,43 @@
+import {h} from 'preact'
+import {useSelector} from '../../../helpers/connect'
+import {getSourceAccountInfo, State} from '../../../state'
+import {AssetFamily, Token} from '../../../types'
+import {assetNameHex2Readable} from '../../../wallet/shelley/helpers/addresses'
+import {AssetItem, Policy} from './sendAdaPage'
+
+const MultiAssetsPage = () => {
+  const {tokenBalance} = useSelector((state: State) => getSourceAccountInfo(state))
+
+  const multiAssets = [
+    ...tokenBalance
+      .sort((a: Token, b: Token) => b.quantity - a.quantity)
+      .map((token: Token) => ({
+        ...token,
+        assetNameHex: token.assetName,
+        assetName: assetNameHex2Readable(token.assetName),
+        type: AssetFamily.TOKEN,
+        star: false,
+      })),
+  ]
+
+  return (
+    <div className="card">
+      <h2 className="card-title">Multi assets</h2>
```

Comment:
We probably want to call it more user friendly, up for discussion. Maybe `Tokens`?

Reply:
Michal suggested "Digital assets"